File: app/src/main/java/seucaioo/com/br/testeretrofit/api/Service.java
package seucaioo.com.br.testeretrofit.api;

import retrofit2.Call;
import retrofit2.http.GET;
import seucaioo.com.br.testeretrofit.model.DatasResponse;

public interface Service {
    @GET("data.json")
    Call<DatasResponse> getDatas();
}

File: app/src/main/java/seucaioo/com/br/testeretrofitkotlin/api/Service.kt
package seucaioo.com.br.testeretrofitkotlin.api

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query
import seucaioo.com.br.testeretrofitkotlin.model.DataResponse

interface Service {

    @GET("data.json")
    fun getDatas() : Call<DataResponse>
}